File: CSC190/exercises/stack.py
class stack:
    def __init__(self):
        self.stck = []

    def put(self,val):
        self.stck = [val] + self.stck
    
    def pop(self):
        temp = self.stck[0]
        self.stck = self.stck[1:]
        return temp



temp = stack()
temp.put(50)
temp.put(30)

print(str(temp.pop()) + " ")
print(temp.pop())

File: CSC190/exercises/stack.c

int main(void){

}    
 
struct stack{
    int *arr;
    int end;   
}

int pop(struct stack *s){
    return (s->arr)[(s->end)];
} 

int push(struct stack *s, int val){
    (s->end) = (s->end) + 1;
    (s->arr)[(s->end)] = val;
    return 0;
}

